src/components/menu/MenuSeparator.tsx

```diff
@@ -1,5 +1,12 @@
 import React from 'react';
+import classNames from 'classnames';
 
-const MenuSeparator = () => <li role="separator" />;
+type MenuSeparatorProps = {
```

Comment:
Sorry I missed this the first time but we need to export this so TypeScript projects can reference the prop types.
```suggestion
export interface MenuSeparatorProps = {
```